lib/otelredis/redis_test.go

```
@@ -0,0 +1,36 @@
+	"github.com/redis/go-redis/v9"
+	"go.opentelemetry.io/otel"
+)
+
+// QueuePush must surface a failed LPUSH as an error rather than reporting
+// success. go-redis stores execution errors on the returned *IntCmd, so a bare
+// (cmd, nil) return would make a connection/context failure look like a
+// successful enqueue — and the recover queue is the sole durable store of a
+// captured ticket's identifiers, so a silent drop can drive a wrongful refund.
+func TestQueuePushPropagatesCommandError(t *testing.T) {
+	rdb := redis.NewClient(&redis.Options{Addr: "127.0.0.1:1"})
+	defer rdb.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `rdb.Close` is not checked

(errcheck)